cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/modeller/model/Task.java

```diff
@@ -59,4 +60,8 @@ public String getRefId() {
         return refId;
     }
 
+    public boolean isAsync() {
+        return preTaskActionData.containsKey(ScoreLangConstants.ASYNC_LOOP_KEY);
```

Comment:
I think it's better passing a boolean to constructor from compiler and adding a field instead of making the pojo familiar with the Lang keys